src/protocols/arweave/scaffold/manifest.js

```diff
@@ -0,0 +1,16 @@
+const source = ({ contract }) => `
+      account: '${contract}'`
```

Comment:
I'm not familiar with arweave terminology, do they use `account` instead of `address` just like Ethereum vs NEAR? 🤔

Reply:
oh it should be `owner` in arweave